### ISSUE TITLE
Error updating Wild Web Developer (eclipse update)

### ISSUE THREAD

**Issue by vanlepthien** [eclipse/wildwebdeveloper#301]
Eclipse update for Wild Web Developer fails:

Wild Web Developer - Developer and debug Web and Node applications	0.7.3.201911200829

An error occurred while collecting items to be installed
session context was:(profile=C__Users_vanle_eclipse_javascript-2019-09_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
No repository found containing: osgi.bundle,org.eclipse.lsp4e,0.12.1.201911131446
No repository found containing: osgi.bundle,org.eclipse.lsp4e.debug,0.11.0.201911212057
No repository found containing: osgi.bundle,org.eclipse.tm4e.core,0.4.0.201911211357
No repository found containing: osgi.bundle,org.eclipse.tm4e.languageconfiguration,0.3.4.201911130739
No repository found containing: osgi.bundle,org.eclipse.tm4e.registry,0.4.0.201911130739
No repository found containing: osgi.bundle,org.eclipse.tm4e.ui,0.4.0.201911211357

Installed plugins:
- Debug Adapter client for Eclipse IDE (Incubation)	0.10.0.201909031808 org.eclipse.lsp4e.debug
- Language Server Protocol client for Eclipse IDE (Incubation)	0.12.0.201909270706	org.eclipse.lsp4e
- TextMate support in Eclipse IDE - Language Configurations	0.3.3.201905072147	org.eclipse.tm4e.languageconfiguration
- TextMate support in Eclipse IDE - Registry management	0.3.4.201909051118	org.eclipse.tm4e.registry
- TextMate support in Eclipse IDE - UI	0.3.4.201909051021	org.eclipse.tm4e.ui
- TextMate support in Java	0.3.4.201909051118	org.eclipse.tm4e.core

Defined repositories:

- "Counterclockwise stable versions Site"	http://updatesite.ccw-ide.org/stable/	Enabled
- 2019-09	http://download.eclipse.org/releases/2019-09	Enabled
- 2019-09	http://download.eclipse.org/releases/2019-09/201909181001	Enabled
- Eclipse Luna repository	http://download.eclipse.org/releases/luna/	Enabled
- Eclipse TeXlipse	http://download.eclipse.org/texlipse/2.0.2/	Enabled
- EPP Marketplace Client	http://download.eclipse.org/mpc/releases/1.7.7	Enabled
- Latest Eclipse Release	http://download.eclipse.org/releases/latest	Disabled
- m2e-apt updates	http://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-apt/	Enabled
- m2e-wtp Mars Release repository	http://download.eclipse.org/m2e-wtp/releases/mars/1.2/	Enabled
- Oomph Latest Milestone	http://download.eclipse.org/oomph/updates/milestone/latest	Enabled
- Oracle Enterprise Pack for Eclipse 12.2.1.8 Dependencies	http://download.oracle.com/otn_software/oepe/12.2.1.8/oxygen/repository/dependencies/	Enabled
- org.eclipse.e4.ui update site	http://download.eclipse.org/e4/snapshots/org.eclipse.e4.ui	Enabled
- org.eclipse.tm4e.repository	http://download.eclipse.org/tm4e/snapshots	Enabled
- repository	http://download.eclipse.org/lsp4e/snapshots	Enabled
- repository	http://download.eclipse.org/wildwebdeveloper/snapshots	Enabled
- The Eclipse Project Updates	http://download.eclipse.org/eclipse/updates/4.13	Enabled
- User Storage All	http://download.eclipse.org/usssdk/updates	Enabled
- User Storage Latest	http://download.eclipse.org/oomph/uss/updates/latest	Enabled
- User Storage Latest Release	http://download.eclipse.org/usssdk/updates/release/latest	Enabled
- 	http://ccw.cgrand.net/updatesite/	Disabled
- 	http://download.eclipse.org/mylyn/releases/3.23	Disabled
- 	http://download.eclipse.org/technology/epp/packages/2019-09/	Enabled
- 	http://master.dl.sourceforge.net/project/sapnweclipse/updates/	Disabled

**Comment by mickaelistria** [comment]
I could reproduce it initially, then tried again with
* http://download.eclipse.org/releases/2019-12
* http://download.eclipse.org/tm4e/snapshots
* http://download.eclipse.org/lsp4e/snapshots
* http://download.eclipse.org/wildwebdeveloper/snapshots
And it all worked for me now.
Can you please try again?

**Comment by vanlepthien** [comment]
I added 2019-12 to my repositories and the update (with all the 2019-12 stuff) failed with the same errors. 
I didn't see the requested version of lsp4e on the lsp4e site, so I'm a bit confused where it's supposed to come from.
I'm going to see if a 2019-12 install fixes things.

**Comment by vanlepthien** [comment]
I looked at the 2019-12 repository, and its lsp4e version is 201910xxxx, so it's old.

When I try to install lsp4e by itself from http://download.eclipse.org/lsp4e/snapshots, I get:

An error occurred while collecting items to be installed
session context was:(profile=C__Users_vanle_eclipse_javascript-2019-09_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
No repository found containing: osgi.bundle,org.eclipse.lsp4e,0.12.1.201911241817

I'm going to try to see why I'm having a problem with the download site

**Comment by vanlepthien** [comment]
I submitted a bug to eclipse org. I don't think this is a Wild Web Developer issue at this point.

https://bugs.eclipse.org/bugs/show_bug.cgi?id=553408

I'll close this when it is confirmed it is not a Wild Web Developer problem.

**Comment by davydnorris** [comment]
I had the same issue, and then was able to install, however Eclipse now runs at 100% CPU every time I start. This is only since the last update

**Comment by vanlepthien** [comment]
2019-12 ships with    
* Wild Web Developer - Web edition in Eclipse IDE (Experimental)	0.4.2.201909061612
The version I have installed (via the snapshot repository)
* Wild Web Developer - Developer and debug Web and Node applications	0.7.2.201911151547
The version that fails installation (from the snapshot) is:
* Wild Web Developer - Developer and debug Web and Node applications	0.7.3.201911200829

**Comment by vanlepthien** [comment]
I think I see what happened. I upgraded Eclipse to 2019-09 before WWD was at a level that supported 2019-09, so I installed the snapshot, which caused problems in its latest incarnation (there may be a problem with the snapshot WWD and lsp4e/tm4e).

I did have the same  problem with a fresh 2019-12 and the snapshot sites, so I did the following.

I removed the snapshot repositories (WWD, lsp4e), found WWD in Marketplace, uninstalled WWD (in the Marketplace dialog) and reinstalled WWD from the Marketplace. Version is now 0.7.1.

I will close this now. Thanks for your help!
